src/controllers/beneficial.owner.type.controller.ts

```diff
@@ -12,35 +12,19 @@ import {
 } from "../model/beneficial.owner.type.model";
 import { isActiveFeature } from '../utils/feature.flag';
 import { getUrlWithParamsToPath } from "../utils/url";
+import { beneficialOwnersTypeEmptyNOCList } from "../validation/async";
```

Comment:
If we're using camel-case, shouldn't this be `beneficialOwnersTypeEmptyNocList` rather than `beneficialOwnersTypeEmptyNOCList`? This is a non-blocking comment so should not hold up merging 🙂

Reply:
NOC is an acronym, not a real word

Reply:
Yes, conventionally, acronyms are also camel-cased.. but it's not a biggie anyway. Will resolve comment